### ISSUE TITLE
Medical Subdepartment Restoration : Virology

### ISSUE THREAD

**Issue by ValiantEsp** [Jonathan-Fort/YWPolarisVore#19]
As it stands, Virology's been mostly shoved into a dark corner and left to weep. There's very little that can actually be done with it, and people don't get viruses naturally, leaving it as a mostly useless part of Medical.

Inspired by TG, the solution is two-fold. One, a rework of virology to be more balanced, and the symptoms more in line with other gameplay effects... (TG currently has a six symptom cap, for instance, and a variety of effects, positive, negative, and neutral to work with.) Adapting that style would give Virology some use outside of the (Admin-only, currently) outbreaks.

And also from TG... Hygiene. Viruses should have a natural way of occurring. The most logical method would be giving it a chance to occur when someone is covered in blood, or in a room with decaying organic matter (I.E, corpses, gibs, no more splattering the area with blood and leaving it to rot.) This, I feel, would give Virology back it's normal use.

And of course, one might ask... "Why?"

Medical has had a few common complaints, but one is that throughout the course of a twenty-hour round, there really isn't that much to do in department. And of course there isn't. Genetics, Virology, both have been stripped down and/or outright ripped out, leaving Medical with little to do unless someone gets hurt. And while 'make your own fun' is of course an understandable answer on an RP server... We're playing a game, not in a chatroom. I feel the restoration of Virology, or perhaps even the implementation of a TG-like system, would not only provide Medical with something to do on those long shifts, but give them yet more reason to interact with the crew. Curing flus, making vaccines, or even making symbiotic viruses to help, as an example, the surface mining crew adapt better to the outside atmosphere.

TL;DR : We have the big room for it, let's make it useful again.

**Comment by ValiantEsp** [comment]
As a supplement, TG's system.
TG allows one to mutate and experiment with a virus to do the following.
A) Isolate symptoms.
B) Neuter symptoms.
C) Mix and match up to six symptoms to create a virus.
D) Randomly create a low-level virus with varying symptoms when someone is filthy/bloody/breathing in corpse gas.
E) Either create vaccines, or create a stronger virus to overwrite the virus someone has.
F) Each symptom has stat thresholds, meaning that the stronger/stealthier/more transmissible/faster acting a virus is, the more severe symptoms may be. This allows for both experimentation, and varying severity in generated viruses.
G) Less instant death syndromes. Yes. TG, of all places, has less symptoms that will outright kill you instantly. Compare to our current code's 'Gibbingtons', 'Monkism', and 'Suicidal' syndromes. Yeah. TG has a symptom that will literally cause you to combust, but there's no 'This virus will gib you, or force you to kill yourself'. Go figure.
